src/main/java/com/bina/varsim/tools/simulation/randVCFgenerator.java

```diff
@@ -82,9 +83,9 @@ public byte sample_genotype(byte geno, Sample_params seen_added, int num_sample,
      * @param geno       allele to be filled in
      */
     public void fillInSeq(Variant var, byte[] insert_seq, int geno) {
-        FlexSeq alt = var.getAlt(geno);
+        Alt alt = var.getAlt(geno);
         if (alt != null) {
-            if (alt.getType() == FlexSeq.Type.INS) {
+            if (FlexSeq.Type.INS.equals(alt.getSeqType())) {
```

Comment:
You are right. For `enum`, `==` and `equals()` are equivalent. I think this is more of my personal preference for comparing objects. I want to avoid `==` except for primitives.